frontend/views/components/StartInviting.vue

```diff
@@ -0,0 +1,66 @@
+<template lang="pug">
+section.card.c-card
```

Comment:
Are we reusing the component outside of the card like in a modal? If so, maybe we should remove the card wrapper here

Reply:
I not forget my comment

Reply:
No, this component is used only once, it's a dashboard widget :)